JavaScript/JavaScript - Queue/queue.js

```diff
@@ -0,0 +1,46 @@
+let Queue = function () {
```

Comment:
Add license header.
With 2020 as the year and your name as copyright owner.

https://www.apache.org/licenses/LICENSE-2.0

Reply:
Added License headers with year and copyright owner name